scrapper/src/main/java/edu/java/controller/ScrapperController.java

```diff
@@ -28,14 +58,16 @@ public class ScrapperController {
             @ApiResponse(responseCode = "400", description = "Invalid request params"),
         }
     )
-    @PostMapping("/tg-chat/{id}")
+    @PostMapping("/user/{id}")
```

Comment:
почитай про правила REST
```suggestion
    @PostMapping("/users/{id}")
```